test/expected/ConvertToBin/MainT.xml

```diff
@@ -0,0 +1,50 @@
+<tokens>
```

Comment:
Do you think we should just remove the tokenizer acceptance tests (and `bin/tokenizer`)? Doesn’t feel like we’re getting much value out of them now that we’ve proved the tokenizer works correctly.

Reply:
I still do get some value looking at the tokens. It's kind of like a high level breakdown of what the source code and grammar breakdown looks like. It'd be even better if it was indented properly. 

To avoid having to add more `.xml` files to appease the tokenizer (obviously adding more files doesn't make the implementation any more correct at this point in time), perhaps I can move the tokenizer-related test files into a separate directory. That way, we can maintain the tokenizer and the tests, and also add more VM files without having to add that extra `.xml` file.

Reply:
Okay, great, if it’s useful then that’s cool.

Reply:
> To avoid having to add more `.xml` files to appease the tokenizer (obviously adding more files doesn't make the implementation any more correct at this point in time), perhaps I can move the tokenizer-related test files into a separate directory.

Alternatively, you could modify the tokenizer acceptance test so that it silently skips `*.jack` files for which the corresponding `*T.xml` file (i.e. `expected_xml_path`) doesn’t exist, rather than failing when it tries to compare the output against a non-existent file.

Reply:
Good idea! https://github.com/wildmaples/jack-compiler/pull/20/commits/58e83528a44ca901ae6190f0934416ca799141eb